Form-Controls/index.html

```diff
@@ -13,15 +13,50 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+       <p>
```

Comment:
I don't think that `p` tag is appropriate here. p is used for paragraphs, but you have form fields. Please, think of updating/removing it.